data/applications/mlpipelines.yaml

```diff
@@ -0,0 +1,17 @@
+metadata:
+  name: mlpipelines
+  annotations:
+    opendatahub.io/categories: 'ML Pipelines'
```

Comment:
Please refer to the list of categories in the README file added in PR#224 to select appropriate categories here. https://github.com/red-hat-data-services/odh-dashboard/pull/224/files#diff-5b17ec93fc259f65542b5ec813578981a6104c4f8238a6006c5cd2a9e92f6786

I think 'Model serving' would be the correct one to use, but let me know if we should add another category to handle this type of application.

Reply:
It would not be model serving.  The main purpose would be experimentation and automation.  I think an argument could be made for categories  'Model training', 'Model development', and 'Model optimization', but we don't really have one for automation.

Reply:
Automation of what, though? Model training and Model optimization make sense to me, but if we need to add Model automation we can do that too. What does "model automation" amount to here?

Reply:
How about something that describes the goals here: https://www.kubeflow.org/docs/components/pipelines/introduction/#what-is-kubeflow-pipelines

Reply:
+1 to `Model training`, `Model development` and `Model optimization`.  Perhaps some of the `Data` categories such as `Data analysis` or `Data preprocessing` as well though though that may tie more directly with the `Model` categories mentioned above.

Just as an FYI, it looks like that README was removed from the `applications` folder at some point, so I was unaware of these category requirements.